build.sh

```diff
@@ -221,20 +225,7 @@ esac
 __BuildOS=$__HostOS
 __BuildType=Debug
 __CMakeArgs=DEBUG
-
-case $__HostOS in
-    FreeBSD)
-        __monoroot=/usr/local
-        ;;
-    OSX)
-        __monoroot=/Library/Frameworks/Mono.framework/Versions/Current
-        ;;
-    *)
-        __monoroot=/usr
-        ;;
-esac
-
-__referenceassemblyroot=$__monoroot/lib/mono/xbuild-frameworks
+__referenceassemblyroot="$(csharp FindMonoXBuildFrameworks.cs)"
```

Comment:
@sokket well, the thing is you can't get the PCL assemblies unless you install the package from the Xamarin repository, which always installs into the paths hardcoded above (in theory you could get the ZIP file from somewhere and unpack it to the right location, but I doubt the average developer knows where to put it).

A **much** better solution would be to just ship the PCL assemblies in a NuGet package (if it's possible to wire that into MSBuild somehow).